net/host.go

```diff
@@ -26,7 +26,7 @@ import (
 const (
 	// ProtocolID is the base atomic swap network protocol ID prefix. The full ID
 	// includes the chain ID at the end.
-	ProtocolID          = "/atomic-swap/0.2"
+	ProtocolID          = "/atomic-swap/0.3"
```

Comment:
is this in anticipation of the release? :p

Reply:
Yes, but going forward we need to update the protocol version every time p2p field names change or when the SwapCreator contract bytecode changes (and this PR does both).